package.json

```diff
@@ -36,13 +36,17 @@
     "validate-commits": "validate-commits"
   },
   "dependencies": {
-    "is-promise": "^2.1.0",
+    "@types/js-cookie": "^2.1.0",
+    "@types/lodash": "^4.14.109",
+    "@types/query-string": "^5.1.0",
     "js-cookie": "^2.1.4",
     "query-string": "^5.0.0",
     "tslib": "^1.8.0"
   },
   "devDependencies": {
     "@bigcommerce/validate-commits": "^2.0.2",
+    "@types/jest": "^22.2.3",
```

Comment:
Can you also install `tslint` and `@bigcommerce/tslint-config`? They should help you pick up some styling issues.